packages/lesswrong/lib/routes.ts

```diff
@@ -111,6 +111,14 @@ lw18ReviewPosts.forEach(
   })
 )
 
+addRoute(
```

Comment:
Sounds like this might need to be forum-gated for now?

Reply:
(Also, if we haven't written the queries to account for AF karma, should consider whether it makes sense to make it accessible on AF or not.)